data-prepper-plugins/common/src/main/java/com/amazon/dataprepper/plugins/source/RandomStringSource.java

```diff
@@ -29,12 +32,14 @@
  * Generates a random string every 500 milliseconds. Intended to be used for testing setups
  */
 @DataPrepperPlugin(name = "random", pluginType = Source.class)
-public class RandomStringSource implements Source<Record<String>> {
+public class RandomStringSource implements Source<Record<Event>> {
 
+    static final String MESSAGE_KEY = "message";
```

Comment:
This came up in another PR. So this constant is now in three locations. 

I think it would be nice for the `Event` to have a builder which is creates a general single item Event.

Something like:

```
Event event = JacksonEvent.fromMessage(UUID.randomUUID().toString());
```

There might be a better name than `fromMessage`.

It could even set the `EventType`.

Reply:
I like the idea. But since it touches the core I will reserve it for a separate PR. Opened an issue for that: #646